cmd/monogen/testapp/wallet/index.html

```diff
@@ -129,7 +129,6 @@ <h2>Metamask Integration</h2>
                         </ul>
                     </li>
                 </ul>
-                <button id="enable-metamask">Add L1 Chain to Metamask</button>
```

Comment:
We'll probably want to update the `Metamask Integration` docs above with a step to reload the page once the wallet was removed if it was previously configured.

Reply:
> if it was previously configured

What do you mean by this?

Reply:
I think I see now.